clicommand/agent_start.go

```diff
@@ -653,6 +654,11 @@ var AgentStartCommand = cli.Command{
 			Usage:  fmt.Sprintf("The behavior when a job is received without a signature. One of: %v. Defaults to %s", verificationFailureBehaviors, agent.VerificationBehaviourBlock),
 			EnvVar: "BUILDKITE_AGENT_JOB_VERIFICATION_NO_SIGNATURE_BEHAVIOR",
 		},
+		cli.StringSliceFlag{
+			Name:   "no-warn-for",
```

Comment:
I can't review the go, but `no-warn-for` feels weird - grammatically incorrect etc. Seems we're okay with slightly longer flags if it means they read more easily. How about `disable-warnings-for`? to match the language you're using in the usage string, and the code (e.g. `DisabledWarningIDs`)

Reply:
yeah, that feels way better